pkg/clients/github/git.go

```diff
@@ -39,12 +39,11 @@ func (g *Github) CreateRef(repository, baseBranchName, sha, newBranchName string
 	if err != nil {
 		return fmt.Errorf("error when creating a new branch '%s' for the repo '%s': %+v", newBranchName, repository, err)
 	}
-	Eventually(func(gomega Gomega) {
-		exist, err := g.ExistsRef(repository, newBranchName)
-		gomega.Expect((err)).NotTo(HaveOccurred())
-		gomega.Expect(exist).To(BeTrue())
-
-	}, 2*time.Minute, 2*time.Second).Should(Succeed()) //Wait for the branch to actually exist
+	//Eventually(func(gomega Gomega) {
```

Comment:
so instead of gomega's `Eventually` func I would use [`WaitUntil` from `util` pkg](https://github.com/redhat-appstudio/e2e-tests/blob/5fc0d3352e338612a4da9bcf9efd6aaecc4e82e3/pkg/utils/util.go#L131)